servers/quarkus-server/src/test/java/org/projectnessie/server/authz/NessieAuthorizationTestProfile.java

```diff
@@ -71,6 +71,9 @@
           .put(
               "nessie.server.authorization.rules.allow_commits_without_entity_changes",
               "op=='COMMIT_CHANGE_AGAINST_REFERENCE' && role=='test_user2' && ref.startsWith('allowedBranch')")
+          .put(
```

Comment:
Needs to be documented on application.properties, too

And probably the nessie-site as well

Reply:
updated the site docs, application.properties and commit kernel doc also.